packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -31,34 +34,43 @@ abstract contract FundingRateApplier is Testable, Lockable {
      * FUNDING RATE APPLIER DATA STRUCTURES *
      ****************************************/
 
-    // Locally stores the finder.
-    // This is private because it is expected that another contract, like the FeePayer, will publicly expose the finder
-    // address.
-    FinderInterface private finder;
-
-    // Last time the `cumulativeFundingRateMultiplier` was updated.
-    uint256 public lastUpdateTime;
+    struct FundingRate {
+        // Current funding rate value.
+        FixedPoint.Signed rate;
+        // Identifier to retrieve the funding rate.
+        bytes32 identifier;
```

Comment:
Can we move the `fundingRateIdentifier` state var to this contract? I know right now its in the `FeePayer` and removing it from there would break tests, but once we deprecate the FRS can we add it here?